+ jpg

GudvinPRO opened the pull request.

GudvinPRO committed:
+ jpg

GudvinPRO committed:
+html

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -0,0 +1,29 @@
+    font-size: 44px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 1.45; /* 145.455% */
```
Коментарі, що вказуються в макеті фігми, потрібно прибирати з коду

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -0,0 +1,29 @@
+}
+
+.testimonials-item {
+    color: var(--primary-text-color);
```
Прописувати color не потрібно, бо колір наслідується із глобальних стилів body, що вказані у файлі styles.css

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -0,0 +1,29 @@
+
+.list {
+    list-style: none;
+}
```
Не потрібно вказувати у вашому css файлі list-style: none; бо стандартні стилі списків вже скинуті у файлі reset.css

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -0,0 +1,29 @@
+
+.staff-block {
+    max-width: 396px;
+}
```
Вираховувати ширину карток потрібно через calc, як ми це робили в домашці: width: calc((100% - 26px) / 3); (прописуємо в .staff-block)

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -0,0 +1,29 @@
+
+.testimonials-staff {
+    display: flex;
+}
```
Списку прописуємо gap: 26px; щоб картки в li стали 396px в ширину (див. коментар до ваших li з класом .staff-block, там має бути ширина вирахувана через calc).

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -0,0 +1,29 @@
+.testimonials-block{
+    max-width: 1240px;
+}
```
Стилі для десктопу 1440px потрібно помістити в медіаправило @media screen and (min-width: 1440px), приберіть max-width, бо в глобальних стилях контейнеру (common.css) вже вказана ширина, вона наслідується. Поверніть діву, що вкладений в секцію, назву класу .container. Також ми починаємо писати стилі з ширини 320px, тобто вони і будуть базовими. Перша точка перелому у нас 375px, тобто потрібен медіазапит @media screen and (min-width: 375px), де ви будете перебивати базові стилі для розміру 320px. Наступна точка перелому для планшетів - @media screen and (min-width: 768px). Далі для десктопу - @media screen and (min-width: 1280px) і @media screen and (min-width: 1440px)

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -0,0 +1,29 @@
+
+.testimonials-item {
+    color: var(--primary-text-color);
+    font-family: Manrope;
```
Приберіть font-family, він наслідується з глобальних стилів

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -0,0 +1,29 @@
+
+.staff-block-text {
+    max-width: 360px;
+}
```
Приберіть ширину і пропишіть падінги за макетом, щоб здвинути контент від країв картки (по 18px праворуч, знизу, ліворуч). В h3 у Вас відсутній марджин між h3 та p, пропишіть марджин, що вказаний в макеті (8px), в клас .staff-title.

GudvinPRO commented on src/partials/testimonials.html:
```diff
@@ -1,3 +1,35 @@
+
+    <ul class="testimonials-staff list">
+        <li class="staff-block">
+            <img src="../img/testimonials/maria.jpg" srcset="../img/testimonials/maria@2.jpg" width="64" alt="Maria">
```
Добре. Зараз все перероблю

VeronikaVaraksina1 reviewed (changes requested):
Потрібні зміни

VeronikaVaraksina1 committed:
Merge branch 'main' into origin/testimonials

GudvinPRO committed:
+html \ css

GudvinPRO committed:
Merge branch 'origin/testimonials' of https://github.com/VeronikaVaraksina1/command-project-vyshyvanka-vibes into origin/testimonials

GudvinPRO committed:
+ сss + @media

GudvinPRO committed:
+html + css

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -1,17 +1,13 @@
-
-    position: relative;
+.testimonials_section{
+    padding-top: 80px;
```
Виправити на padding-bottom: 80px; щоб забезпечити правильний відступ від вашої секції до футеру

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -61,6 +57,10 @@
 
 @media screen and (min-width: 768px) {
+    .testimonials_section{
+        padding-top: 100px;
```
Виправити на padding-bottom: 100px; щоб забезпечити правильний відступ від вашої секції до футеру

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -87,6 +87,10 @@
 
 @media screen and (min-width: 1280px) {
+    .testimonials_section{
+        padding-top: 150px;
```
Виправити на padding-bottom: 150px; щоб забезпечити правильний відступ від вашої секції до футеру

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -1,17 +1,13 @@
 
-
-.container_image {
```
Назви класів пишемо через дефіс, виправте на .container-image

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -1,17 +1,13 @@
-    height: 34px;
-
-    position: relative;
+.testimonials_section{
```
Назви класів пишемо через дефіс, виправте на .testimonials-section

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -1,17 +1,13 @@
+    padding-top: 80px;
 }
 
 .image_icon {
```
Назви класів пишемо через дефіс

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -61,6 +57,10 @@
 }
 
 @media screen and (min-width: 768px) {
+    .testimonials_section{
```
Назви класів пишемо через дефіс

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -61,6 +57,10 @@
+        padding-top: 100px;
+    }
+
     .testimonials-item {
```
В стилях h2 виправте кому на крапку в правилах, ось так щоб було:
line-height: 1.38;
letter-spacing: -0.03em;

В планшетних та десктопних стилях ви також неправильно вказали (має бути з мінусом), ось так треба: letter-spacing: -0.03em;

VeronikaVaraksina1 commented on src/partials/testimonials.html:
```diff
@@ -1,14 +1,13 @@
+            
             <div class="staff-block-text">
                 <h3 class="staff-title">Maria Tkachuk</h3>
                 <p class="staff-paragraph">
```
В стилях параграфу приберіть font-style: normal; він іде за замовчуванням з глобальних стилів

GudvinPRO replied:
зробив

VeronikaVaraksina1 commented on src/css/testimonials.css:
```diff
@@ -41,7 +38,6 @@
 }
 
-
 .staff-title {
```
Приберіть зі стилів h3 font-style: normal; він йде за замовчуванням з глобальних стилів

GudvinPRO replied:
Зробив

VeronikaVaraksina1 commented on src/partials/testimonials.html:
```diff
@@ -1,14 +1,13 @@
 
     <ul class="testimonials-staff">
 
         <li class="staff-block">
```
В стилях .testimonials-staff на десктопі 1280px calc має бути width: calc((100% - 26px) / 3); (значення, що віднімається від 100% розміру батьківського елементу, має дорівнювати gap, що вказаний в .testimonials-staff, а саме 26px)

VeronikaVaraksina1 commented on src/partials/testimonials.html:
```diff
@@ -1,14 +1,13 @@
-<section class="section">
+<section class="section testimonials_section">
```
Назву класів ми пишемо через дефіс, виправіть на testimonials-section

VeronikaVaraksina1 commented on src/partials/testimonials.html:
```diff
@@ -18,9 +17,8 @@ <h3 class="staff-title">Maria Tkachuk</h3>
-              <img class="image_icon" src="../img/testimonials/sergey.png" srcset="../img/testimonials/sergey@2.png" width="64" alt="Sergey">
-            </div>
+          
+            <img class="image_icon" src="../img/testimonials/sergey.png" srcset="../img/testimonials/sergey@2.png" width="64" alt="Sergey">
```
Назву класу пишемо через дефіс

Не підтягується зображення для щільності пікселів DPR: 1.0
Ви забули вказати дескриптор 2x в атрибуті srcset. В слак надам Вам скріншот, як це має виглядати

VeronikaVaraksina1 commented on src/partials/testimonials.html:
```diff
@@ -31,9 +29,8 @@ <h3 class="staff-title">Sergey Rybachok</h3>
-              <img class="image_icon" src="../img/testimonials/natalia.png" srcset="../img/testimonials/natalia@2.png" width="64" alt="Natalia">
-            </div>
+            
+            <img class="image_icon" src="../img/testimonials/natalia.png" srcset="../img/testimonials/natalia@2.png" width="64" alt="Natalia">   
```
Назву класу пишемо через дефіс

Не підтягується зображення для щільності пікселів DPR: 1.0
Ви забули вказати дескриптор 2x в атрибуті srcset. В слак надам Вам скріншот, як це має виглядати

VeronikaVaraksina1 commented on src/partials/testimonials.html:
```diff
@@ -1,14 +1,13 @@
-            </div>
-
+            
+            <img class="image_icon" src="../img/testimonials/maria.png" srcset="../img/testimonials/maria@2.png" width="64" alt="Maria">
```
Назву класу пишемо через дефіс

Не підтягується зображення для щільності пікселів DPR: 1.0
Ви забули вказати дескриптор 2x в атрибуті srcset. В слак надам Вам скріншот, як це має виглядати

GudvinPRO committed:
1

GudvinPRO committed:
Правки

Виправиви падінг
Виправив дефіси
Виправив діспл.\флекс
І все одно гуляє 2 пикселі по висоті)

GudvinPRO committed:
1

GudvinPRO committed:
1
